actions/contacts/models.go

```diff
@@ -15,6 +15,16 @@ type UpsertContact struct {
 	RequesterPaymail string `json:"requesterPaymail"`
 }
 
+// ContactData is a type for contact data
+type ContactData struct {
+	// OwnerXpubID is a xpub id related to contact.
+	OwnerXpubID string `json:"xpub_id"`
```

Comment:
I think in new admin api we moved from snake case to camel case? I might be wrong but shouldn't there be xpubId and fullName or something in json struct?